pkg/virtctl/credentials/addkey/addkey_test.go

```diff
@@ -291,7 +294,7 @@ var _ = Describe("Credentials add-ssh-key", func() {
 		const testKey = "key contents in file"
 
 		filename := filepath.Join(GinkgoT().TempDir(), "test-key-file")
-		Expect(os.WriteFile(filename, []byte(testKey), 0666)).To(Succeed())
+		Expect(os.WriteFile(filename, []byte(testKey), 0600)).To(Succeed())
```

Comment:
How is this change related to the linter?

Reply:
It was checked by the linter. I got this error message:
```
pkg/virtctl/credentials/addkey/addkey_test.go:297:10: G306: Expect WriteFile permissions to be 0600 or less (gosec)
		Expect(os.WriteFile(filename, []byte(testKey), 0666)).To(Succeed())
		       ^
```

Reply:
Interesting. Thanks for sharing.
Perhaps you can add it to the commit message, it is not that trivial.

Reply:
done